### PR TITLE
Fix several typos

### PR DESCRIPTION
**Pull request by jbzdarkid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,15 +146,15 @@ Explaining what each part does:
...
-    used launch a game and give time to select a region or other unsuported options.
+    used launch a game and give time to select a region or other unsupported options.
   * `notask` doesn't start the client trough a scheduled task, this will make the steam overlay also apply to the battle.net client
   * `noadmin` tries to apply compatibility flags to the game to avoid calling the UAC, this is an untested hack that can break the game
   **do not use unless you know what you're doing**.
 
 ## Special Thanks
 
 * internet coder Maruf for ghost tray icon fix code
-* github Ethan-BB for the new parameters to launch games on battle.net
```
Comment (dafzor) [inline README.md]: github here is to identify the platform of the username, it should not be removed.

File: README.md [hunk 2]
```diff
@@ -145,15 +146,15 @@ Explaining what each part does:
...
+* Ethan-BB for the new parameters to launch games on battle.net
 * github RobFreiburger and iMintty for Starcraft Remastered and Destiny 2 support respectively.
 * /u/fivetwofoureight for creating and allowing me to use his icon.
 * /u/malecden, Maverick, /u/sumphatguy and others for their help pointing out bugs.
```
Comment (dafzor) [inline README.md:160]: feel free to add yourself at the end for spellchecking.

**Review by dafzor — CHANGES_REQUESTED** [review]
Just two changes, otherwise looking good

File: README.md [hunk 3]
```diff
@@ -145,15 +146,15 @@ Explaining what each part does:
...
+    used launch a game and give time to select a region or other unsupported options.
   * `notask` doesn't start the client trough a scheduled task, this will make the steam overlay also apply to the battle.net client
   * `noadmin` tries to apply compatibility flags to the game to avoid calling the UAC, this is an untested hack that can break the game
   **do not use unless you know what you're doing**.
 
 ## Special Thanks
 
 * internet coder Maruf for ghost tray icon fix code
-* github Ethan-BB for the new parameters to launch games on battle.net
+* Ethan-BB for the new parameters to launch games on battle.net
```
Comment (jbzdarkid) [inline README.md]:
```suggestion
* github Ethan-BB for the new parameters to launch games on battle.net
```

File: README.md (hunk 2 above)
Comment (jbzdarkid) [inline README.md:160]:
```suggestion
* /u/malecden, Maverick, /u/sumphatguy and others for their help pointing out bugs.
* github jbzdarkid for fixing some typos in the documentation.
```

**Review by dafzor — APPROVED** [review]
all good, thanks for the fixes.

**Pull request merged**